### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by gary-host-laptop** [Colllect/Colllect#84]
Title

**Comment by lostdesign** [comment]
I am also following this project for a while. I simply assume this is a side-project, as you can see by the commits (https://github.com/Colllect/Colllect/commit/228a271944e844f5b931527b4635414ed99580ff) it still has some traffic.

**Comment by Alex-D** [comment]
Hi!

Colllect is paused yet.

To be honest, I've started Colllect as a side-project, but it will engage some cost, so I've worked on a business plan and a way to not lose money, and maybe make it a part of my incomes.

The current status is: MVP API is done, the front is still WIP, form components are ready, but the UI is not usable for now.

I've worked on some other projects:

- [Trumbowyg](https://alex-d.github.io/Trumbowyg/) (3400 ★) — A lightweight HTML WYSIWYG editor (8kB)
- [Monitoror](https://monitoror.com) (3100 ★) — Unified monitoring wallboard
- [Column View](https://column-view.com) — Display multiple web pages simultaneously

I'm currently building the v3 of Trumbowyg which will be used to edit notes in Colllect. [See more about Trumbowyg v3](https://github.com/Alex-D/Trumbowyg/issues/875).

Also, I'm a full-time freelancer now, so I did not have enough time to finish a Colllect alpha for now, but I did not abandon this project.

Thank you for your interest in Colllect :)

**Comment by gary-host-laptop** [comment]
Yeah, I understand the difficulties with money and the absence of time to work on the project, no hurries. I also checked your other projects but they seem focused more on people who program, and I am not one of them.

I really hope you do not abandon this, it's the only software out there which is libre and aesthetic, the rest seem outdated, ugly, and a privacy nightmare. I wish you the best and I hope you have the time to finish it, and if you ever need some early testers, I can help with that.

**Comment by lostdesign** [comment]
If I wasn't packed with side projects and work, I'd love to help out on this one. Something like that is still much needed, raindrop isn't for me and I simply love the fact that this project is OSS. 2017 we also started a bookmarking app that failed, team vanished. So maybe I can invest myself in here at some point. The issues seem rather well detailed.

**Comment by Alex-D** [comment]
> raindrop isn't for me

What makes you think Colllect can be better?

**Comment by lostdesign** [comment]
The design alone and it being OSS is enough. Everything else can be fixed while working on such project. I wish the dev of raindrop all the success, but I simply disagree with their pricing. Hence why it's not for me, back then the nested grouping was a premium feature and the only one that I needed. I don't mind paying for services at all, back then it didn't feel right.

Colllect is much more broad than raindrop too. I love the fact that you can also sort and maintain your UI inspirations. Raindrop is limited to bookmarks IIRC, Colllect offers notes/links and and pictures - dearly need that ^^

**Comment by GoodM4ven** [comment]
I wonder how you manage bookmarks currently, while there isn't a Colllect yet! @Alex-D

Also, it feels sad that such a beautiful landing page is for a _dead_ project!
https://colllect.io/
